### PR TITLE
[topgen] Fix importing bug

### PR DESCRIPTION
**Pull request by eunchan**
topgen imports object file with module's name not type.
It creates empty object which results in key error.

### CODE REVIEW

File: util/topgen.py [hunk 1]
```diff
@@ -730,7 +730,7 @@ def main():
 
         # Create filtered list
         filter_list = [
-            module['name'] for module in topcfg['module']
+            module['type'] for module in topcfg['module']
```
Comment (tjaychen) [inline util/topgen.py:733]: oops! sorry, i'm pretty sure I created this bug. 
How did this manifest btw?

Comment (eunchan) [reply util/topgen.py:733]: I found this as I put the `_aon` suffix to those modules in AON domain :)

Comment (eunchan) [reply util/topgen.py:733]: I think I made this bug. though...

File: util/topgen/merge.py [hunk 2]
```diff
@@ -574,6 +575,9 @@ def amend_pinmux_io(top):
         # Parse how many signals
         m = lib.get_module_by_name(top, mname)
 
+        if m is None:
+            log.error("Module {} is not searchable.".format(mname))
```
Comment (imphil) [inline util/topgen/merge.py:579]: You use `m` in the statement below; does it copy with `m` being `None`, or are you missing a return or else here?

**Pull request merged**